pkg/cmd/importcmd/import.go

```diff
@@ -1,6 +1,7 @@
 package importcmd
 
 import (
+	goerrors "errors"
```

Comment:
Why do we call this `goerrors`, why not use the default name `errors`?

Reply:
because `github.com/pkg/errors` is also imported... hm, maybe I replace its occurrences with `errors` now that we updated to go 1.13...

Reply:
äh, no, can't do that quickly since callers might rely on them and `github.com/pkg/errors` v0.8.1 is not compatible with the new go `errors`...
Therefore I could try a) update the pkg/errors dependency to 0.9 which supports go 1.13 errors or b) continue to use pkg/errors only.
WDYT? @mrageh

Reply:
well i am not taking the risk for single line ;). I changed it to use `github.com/pkg/errors`.